README.md

```diff
@@ -170,6 +170,76 @@ Now proceed with the following steps:
 Now you are ready to start configuring your network! **If you need help** you can ask questions
 on one of the official [OpenWISP Support Channels](http://openwisp.org/support.html).
 
+Install OpenWISP for Development Environment
+-------------------------------------------------
+First time create directory where you want to place OpenWISP2 configuration.
+
+For example:
+```bash
+    $ mkdir ~/openwisp-dev
+    $ cd ~/openwisp-dev
+```
+
+Create directory roles.
+```bash
+    $ mkdir roles
+    $ cd roles
+```
+Clone `ansible-openwisp2` and `Shouts.postfix` repository to your roles folder, you will see `openwisp.openwisp2` and `Stouts.postfix` folder. 
+```bash
+    $ git clone https://github.com/openwisp/ansible-openwisp2.git openwisp.openwisp2
+    $ git clone https://github.com/nemesisdesign/Stouts.postfix
+```
+
+After cloning in the folder roles, go one directory up (back to the location where the roles folder is found), Create hosts file and playbook.yml with:
+```bash
+    $ touch hosts
+    $ touch playbook.yml
+```
+
+Next, write hosts file with your local machine. For example we using localhost with ip 127.0.0.1
```

Comment:
from here on the text is redundant and duplicated.

Conclude the section with something like:

```
From here on you can follow the instructions available at the following sections:

- [#create-inventory-file](Create inventory file)
- [#create-playbook-file](Create playbook file)
- [#run-the-playbook](Run the playbook)

You can remove the rest of the text.
```